testscommon/shardedDataStub.go

```diff
@@ -29,63 +30,63 @@ func NewShardedDataStub() *ShardedDataStub {
 }
 
 // RegisterOnAdded -
-func (shardedData *ShardedDataStub) RegisterOnAdded(handler func(key []byte, value interface{})) {
-	if shardedData.RegisterOnAddedCalled != nil {
-		shardedData.RegisterOnAddedCalled(handler)
+func (sd *ShardedDataStub) RegisterOnAdded(handler func(key []byte, value interface{})) {
```

Comment:
for all stub, I name the receiver pointer as `stub`
So this will yield as `func (stub *ShardedDataStub) RegisterOnAdded(handler func(key []byte, value interface{})) {`
This name will match each and every stub we create :D

Reply:
interesting approach. however, it could come in a future PR that changes all stubs, so I wouldn't change this in this PR

Reply:
right, for all new stubs I do it like this.